packages/csv-parser-price/src/constants.js

```diff
@@ -0,0 +1,27 @@
+const CONSTANTS = {
+  host: {
+    api: 'https://api.sphere.io',
+    auth: 'https://auth.sphere.io',
+  },
+
+  field: {
+    integer: new RegExp(/^-?\d+$/),
+    money: new RegExp(/^([A-Z]{3}) (-?\d+)$/),
+  },
+
+  header: {
+    sku: 'variant-sku',
+  },
+
+  standardOption: {
+    batchSize: 100,
+    delimiter: ',',
+  },
+}
+
+// Go through object because `freeze` works shallow
+Object.keys(CONSTANTS).forEach((key) => {
```

Comment:
My inner spirit tells me there is a module that does it, so should be refactored to a module if non exist 

👻